### PR TITLE
ramips: fix Lenovo Y1/Y1s led in init script

### PR DESCRIPTION
**Pull request by hbl0307106015**
This commit fix the y1 & y1s leds' definition.

### CODE REVIEW

File: target/linux/ramips/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -335,11 +335,19 @@ wndr3700v5)
 wt3020)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
 	;;
-y1|\
+y1)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
```
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: This shouldn't be required. The LED is already handled by target/linux/ramips/base-files/etc/diag.sh.

File: target/linux/ramips/base-files/etc/board.d/01_leds [hunk 2]
```diff
@@ -335,11 +335,19 @@ wndr3700v5)
...
+	set_usb_led "$board:blue:usb"
+	set_wifi_led "$board:blue:wifi" "wlan1"
+	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wifi5g" "wlan0" "link tx rx"
+	ucidef_set_led_netdev "lan" "lan" "$board:blue:lan" "eth0.1" "tx rx"
+	ucidef_set_led_netdev "wan" "wan" "$board:blue:internet" "eth0.2" "tx rx"
+	;;
 y1s)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
```
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: This shouldn't be required. The LED is already handled by target/linux/ramips/base-files/etc/diag.sh.

File: target/linux/ramips/base-files/etc/board.d/01_leds [hunk 3]
```diff
@@ -335,11 +335,19 @@ wndr3700v5)
 wt3020)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
 	;;
-y1|\
+y1)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	set_usb_led "$board:blue:usb"
+	set_wifi_led "$board:blue:wifi" "wlan1"
```
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: use ucidef_set_led_netdev for the 2.4 GHZ as well. Don't different functions for 2.4 and 5 GHz wifi. It is confusing.

File: target/linux/ramips/base-files/etc/board.d/01_leds [hunk 4]
```diff
@@ -335,11 +335,19 @@ wndr3700v5)
...
+	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wifi5g" "wlan0" "link tx rx"
```
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: drop the "link tx rx" part. It's the default if nothing is specified.

File: target/linux/ramips/base-files/etc/board.d/01_leds [hunk 5]
```diff
@@ -335,11 +335,19 @@ wndr3700v5)
...
+	ucidef_set_led_netdev "lan" "lan" "$board:blue:lan" "eth0.1" "tx rx"
+	ucidef_set_led_netdev "wan" "wan" "$board:blue:internet" "eth0.2" "tx rx"
```
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: any reason why the leds shouldn't show the link state?

**Comment by mkresin** [comment]
Please use the full board name in the commit title.

Please add a commit message to your commit. It should describe why you have done something.

File: target/linux/ramips/base-files/etc/board.d/01_leds (hunk 5 above)
Comment (hbl0307106015) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: For the LAN and WAN port, many of our customers think the LED should turn off after they pull out the cable."tx rx" state do the job and this really avoid confusing them.

File: target/linux/ramips/base-files/etc/board.d/01_leds (hunk 3 above)
Comment (hbl0307106015) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: ok, I will use ucidef_set_led_netdev instead

File: target/linux/ramips/base-files/etc/board.d/01_leds (hunk 5 above)
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: Well, that sounds exactly what the ```link``` trigger should do.

**Comment by hbl0307106015** [comment]
@mkresin  The PR has been changed, please check it again. Thanks!

File: target/linux/ramips/base-files/etc/board.d/01_leds (hunk 5 above)
Comment (mkresin) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: Gave it a quick test on a VGV7510KW22 and it works for me with the link parameter.

If I unplug the network cable:

```
[  213.263560] eth0: port 3 lost link
[  213.266285] br-lan: port 1(eth0.1) entered disabled state
```

And the configured LED is switched off.

There must be something else wrong on the Y1/Y1s.

**Comment by blogic** [comment]
merged, Thanks !

**Pull request closed without merging**